packages/website/src/features/Search/PackageCard/PackageCardExpandable.tsx

```diff
@@ -79,7 +79,10 @@ export const PackageCardExpandable: FC<IPackageCardProps> = ({
           animate={{ height: 'auto' }}
           className="overflow-hidden bg-background"
         >
-          <div className="align-middle overflow-auto" style={{ maxHeight }}>
+          <div
+            className="align-middle overflow-auto max-h-[280px]"
```

Comment:
```suggestion
            className="align-middle overflow-auto max-h-[290px]"
```

make the next row more obvious